File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webcountry.ts
import { EmailDistributionList } from './generated/EmailDistributionList.model';
import { EmailDistributionListContact } from './generated/EmailDistributionListContact.model';
import { EmailDistributionListContactLanguage } from './generated/EmailDistributionListContactLanguage.model';
import { EmailDistributionListLanguage } from './generated/EmailDistributionListLanguage.model';
import { WebBase } from './webbase';

export class WebCountry extends WebBase {
    TVItemParentList: WebBase[] = [];
    TVItemProvinceList: WebBase[] = [];
    EmailDistributionListList: EmailDistributionList[] = [];
    EmailDistributionListLanguageList: EmailDistributionListLanguage[] = [];
    EmailDistributionListContactList: EmailDistributionListContact[] = [];
    EmailDistributionListContactLanguageList: EmailDistributionListContactLanguage[] = [];
}

File: src/dlls/CSSPModels_backup/srcweb/WebCountry.cs
﻿/*
 * Manually edited
 * 
 */
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPModels
{
    [NotMapped]
    public partial class WebCountry : WebBase
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<WebBase> TVItemProvinceList { get; set; }
        public List<EmailDistributionList> EmailDistributionListList { get; set; }
        public List<EmailDistributionListLanguage> EmailDistributionListLanguageList { get; set; }
        public List<EmailDistributionListContact> EmailDistributionListContactList { get; set; }
        public List<EmailDistributionListContactLanguage> EmailDistributionListContactLanguageList { get; set; }
        #endregion Properties

        #region Constructors
        public WebCountry()
        {
            TVItemParentList = new List<WebBase>();
            TVItemProvinceList = new List<WebBase>();
            EmailDistributionListList = new List<EmailDistributionList>();
            EmailDistributionListLanguageList = new List<EmailDistributionListLanguage>();
            EmailDistributionListContactList = new List<EmailDistributionListContact>();
            EmailDistributionListContactLanguageList = new List<EmailDistributionListContactLanguage>();
        }
        #endregion Constructors
    }
}
